### PR TITLE
367  Fixing spacing on start case form 

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: web-client/src/views/StartCase.jsx [hunk 1]
```diff
@@ -162,39 +169,41 @@ export const StartCase = connect(
                   validateStartCaseSequence();
                 }}
               />
-              <div className="usa-input-error-message">
-                {validationErrors.stinFile}
-              </div>
+              {validationErrors.stinFile && (
```
Comment (rachaelparris) [inline web-client/src/views/StartCase.jsx:172]: Is this to remove the extra space generated by the empty error message div? Maybe need this fix also for the `validationErrors.hasIrsNotice` and `validationErrors.procedureType`?

Comment (ericsorenson) [reply web-client/src/views/StartCase.jsx:172]: @rachaelparris you're exactly right, I should have mentioned in the PR that I'm trying to make this a manageable size, so I didn't do it all at once. Those are next up :-)

Comment (rachaelparris) [reply web-client/src/views/StartCase.jsx:172]: Gotcha!

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/index.pug [hunk 2]
```diff
@@ -5,6 +5,11 @@ html(lang="en")
     meta(built="" + Date())
     meta(name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no")
     title U.S. Tax Court Electronic Filing and Case Management System
+    link(rel="preload" as="font" type="font/woff2" href="fonts/noto-serif-jp-v4-latin-400.woff2")
```
Comment (codyseibert) [inline web-client/src/index.pug:8]: what is this stuff?

Comment (ericsorenson) [reply web-client/src/index.pug:8]: It preloads he custom fonts before the CSS is parsed

**Review by wbyoko — APPROVED** [review]

**Pull request merged**